### DISCUSSION TITLE
Unable to watch the rtmp stream being pushed

### DISCUSSION THREAD

**Discussion by javirescalvo** [ant-media/Ant-Media-Server#5118] [Q&A]
Hi! I cannot see the stream I'm pushing to rtmp. I've tried creating a stream from a VOD as this user suggested: https://community.antmedia.io/topic/1688-can%C2%B4t-stream-to-my-own-website-broadcast-does-not-start/#comment-6398

But still no luck. I've checked open ports, reinstalled a few times and still nothing shows up...

**Comment by yashtandon113** [comment]
@javirescalvo
If you are using Enterprise Edition then make sure that UDP ports 50000-60000 are enabled on your firewall to play the stream with WebRTC and if you are using the Community edition then HLS should play without any problem.

Also check the below document for RTMP publishing.
https://antmedia.io/docs/category/rtmp/

**Reply by javirescalvo** [reply]
Thank you! I've found out about the playOrder=hls parameter...
